check-ecobalyse-data-sync.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -euo pipefail
+
+# Directory where files will be downloaded
+TEMP_DIR=$(mktemp -d)
+ECOBALYSE_DATA_BRANCH="${ECOBALYSE_DATA_BRANCH:-main}"
+RAW_GITHUB_URL="https://raw.githubusercontent.com/MTES-MCT/ecobalyse-data/${ECOBALYSE_DATA_BRANCH}"
+
+# Cleanup function
+cleanup() {
+    rm -rf "$TEMP_DIR"
+}
+trap cleanup EXIT
+
+# Files to check
+FILES=(
+    "public/data/food/ingredients.json"
+    "public/data/food/processes.json"
+    "public/data/textile/materials.json"
+    "public/data/textile/processes.json"
+    "public/data/object/processes.json"
+)
+
+echo "Downloading files from ecobalyse-data repository (branch: ${ECOBALYSE_DATA_BRANCH})..."
+
+# Create necessary directories in TEMP_DIR
+for file in "${FILES[@]}"; do
+    mkdir -p "$TEMP_DIR/$(dirname "$file")"
+done
+
+# Download each file
+for file in "${FILES[@]}"; do
+    wget -q "$RAW_GITHUB_URL/$file" -O "$TEMP_DIR/$file" || {
+        echo "--⚠️  Failed to download $file from ecobalyse-data repository"
+        DIFFERENCES_FOUND=1
+        continue
+    }
+done
+
+echo "Comparing JSON files between ecobalyse-data and ecobalyse repositories..."
+DIFFERENCES_FOUND=0
```

Comment:
As already mentioned, this should be defined earlier in the file

Reply:
done